### ISSUE TITLE
Learning rate tuning

### ISSUE THREAD

**Issue by USBperipheral** [snucml/LargeLossMatters#6]
Hi,
in the paper supplementary you mention you "search the learning rate by dividing the range between values in {0.01, 0.001, 0.0001, 0.00001} into quarters". Does this mean you search over learning rates 0.01, 0.0075, 0.050, 0.025, 0.01 etc? Over all learning rates, do you pick the run with the best validation accuracy and run that one once on the test set? Or do you calculate the test set result for each learning rate and report the best value?

Thanks!

**Comment by youngwk** [comment]
Hi, thanks for your attention in our work.

Yes the entire search space of learning rate is {1e-5, 2.5e-5, 5e-5, 7.5e-5, 1e-4, 2.5e-4, 5e-4, 7.5e-4, 1e-3, 2.5e-3, 5e-3, 7.5e-3, 1e-2}.

About reporting test set performance, we used the latter approach. However, after checking results we found that hyperparameters that perform best in the test set tend to perform best in the val set also. So there is no significant difference in the final result whether the former method is used or the latter method is used.
